Training25/Program.cs

```diff
@@ -3,11 +3,38 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program to check whether the password is strong or not with some criteria.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
+
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      while (true) {
+         Write ("Strong Password\n~~~~~~~~~~~~~~~\nEnter a password: ");
+         var password = ReadLine ();
+         if (string.IsNullOrWhiteSpace (password)) { WriteLine ("Invalid Input!\n"); continue; }
+         bool iWeak = false;
+         if (password.Length < 6) iWeak = PrintError ("Must be atleast 6 characters in length");
+         if (!password.Any (char.IsDigit)) iWeak = PrintError ("Must conatin atleast one digit");

```

Comment:
```suggestion
         if (!password.Any (char.IsDigit)) iWeak = PrintError ("Must contain atleast one digit");
```
Turn on the spell checker, you will see the markers as shown below:

<img width="505" height="245" alt="Image" src="https://github.com/user-attachments/assets/9caffecd-1ccd-4437-9da5-16bfc4be8143" />